docs: Update the extensibility docs (#3582)
kyma-project/busola: docs/extensibility/140-static-extensions.md

# Configure a Config Map for Static Extensions

> You can define a static extension by adding the `busola.io/extension:statics` label to the ConfigMap. You don't need the **general** section as static extensions present data that are not connected to any resource. Instead, they may use information from the page they are displayed on using the `$embedResource` variable .You can provide all the ConfigMap data sections as either JSON or YAML.

Reviewer: ```suggestion
You can define a static extension by adding the `busola.io/extension:statics` label to the ConfigMap. You don't need the **general** section as static extensions present data that are not connected to any resource. Instead, they may use information from the page they are displayed on using the `$embedResource` variable. You can provide all the ConfigMap data sections as either JSON or YAML.
```

| **externalNodes.icon**           | No       | string                                       | An icon that you can choose from the [Icon Explorer](https://sdk.openui5.org/test-resources/sap/m/demokit/iconExplorer/webapp/index.html#/overview). |
| **externalNodes.children**       | No       | string                                       | a list of child Nodes containing details about the links.                                                                                            |
| **externalNodes.children.label** | No       | string                                       | a displayed label.                                                                                                                                   |
| **externalNodes.children.link**  | No       | string, [JSONata](100-jsonata.md) expression | a link to an external website.                                                                                                                       |

### Example

```yaml
general:
  externalNodes:
    - category: My Category
      icon: course-book
      children:
        - label: Example Node Label
          link: 'https://github.com/kyma-project/busola'
```

> To see an exemplary configuration of the `External Nodes` feature in static extensions, check the [configuration example](examples/../../../examples/statics/statics-external-nodes.yaml).

Reviewer: ```suggestion
For more information on an exemplary configuration of the `External Nodes` feature in static extensions, see the [configuration example](examples/../../../examples/statics/statics-external-nodes.yaml).
```
